src/index.html

```diff
@@ -15,16 +15,18 @@
       </div>
       <ul>
         <li>Meals</li>
```

Comment:
- [ ] The Counters should be present in all the interfaces that show the number of items (home), and the number of comments (comments popup), good job implementing the center for the comments however I can't see the center for the items in the home page, kindly make sure to include this in your code and keep in mind that each counter should be implemented as a separate module

![Screenshot_20230106_182126](https://user-images.githubusercontent.com/25227022/211064325-ff416215-4b44-439f-a829-4154f82b74f5.png)